src/components/ControlGroup/ControlGroup.spec.tsx

```diff
@@ -0,0 +1,27 @@
+import { describe, expect, it } from "bun:test";
+import { render, screen } from "@testing-library/react";
+import { fireEvent } from "@testing-library/react";
+import ControlGroup from "./ControlGroup";
+import { Control } from "@/components/Docs/Control/Control";
+import { BooleanControl } from "@/components/Docs/Control/components/BooleanControl/BooleanControl";
+
+describe("Control Group", () => {
+	it("Open Close Accordion", async () => {
```

Comment:
Follow `BDD testing naming/implementation ` please it's what we'll be doing project wide and want the project to read as if it was written by a single person.

In this case test is fine name just needs to be changed from `Open Close Accordion` to...

`Should show element when closed and clicked`
or
follow gherkin style `Given` `When `Then` language